cluster.yml

```diff
@@ -86,19 +97,25 @@
 
 - hosts: calico-rr
   any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
+  vars_files:
+    - roles/download/defaults/main.yml
   roles:
     - { role: kubespray-defaults}
     - { role: network_plugin/calico/rr, tags: network }
 
 - hosts: k8s-cluster
   any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
+  vars_files:
+    - roles/download/defaults/main.yml
   roles:
     - { role: kubespray-defaults}
     - { role: dnsmasq, when: "dns_mode == 'dnsmasq_kubedns'", tags: dnsmasq }
     - { role: kubernetes/preinstall, when: "dns_mode != 'none' and resolvconf_mode == 'host_resolvconf'", tags: resolvconf }
 
 - hosts: kube-master[0]
   any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
+  vars_files:
```

Comment:
LGTM for WIP although there are more playbooks to add these

Reply:
I hate this solution. Therefore WIP. I will try to experiment with some other options to get this achieved without duplicating data.